runtime/ftplugin/mediawiki.vim

```diff
@@ -41,5 +37,5 @@ endif
 setlocal foldexpr=getline(v:lnum)=~'^\\(=\\+\\)[^=]\\+\\1\\(\\s*<!--.*-->\\)\\=\\s*$'?\">\".(len(matchstr(getline(v:lnum),'^=\\+'))-1):\"=\"
 setlocal fdm=expr
 
-let b:undo_ftplugin += "setl commentstring< comments< formatoptions< foldexpr< fdm< fileencoding<"
-let b:undo_ftplugin += " matchpairs< formatoptions< linebreak< wrap< textwidth<"
+let b:undo_ftplugin = "setl commentstring< comments< formatoptions< foldexpr< fdm<"
```

Comment:
Sorry I just noticed. We reset `'commentstring'` option, but never set it. I can remove this while merging, or do we want to explicitly also set the 'commentstring' option for mediawiki?